OPI_RELEASE_APPROACH.md

```diff
@@ -0,0 +1,85 @@
+# OPI Release Approach
+
+OPI will release some artifacts around the work being done in the
```

Comment:
the release cadence is TBD ?

Reply:
I mean, since so far there are like 3 of us writing code, it's unclear how sustainable anything other than *an initial release* is. But sure, I'll add some notes about a release cadence.